server/helpers/planner_helper/planner_helper_data_types.py

```diff
@@ -6,7 +6,26 @@
 from helpers.common_helper.hash_helper import get_list_hash
 from helpers.nl2plan_helper.nl2ltl_helper import LTLFormula, CachedPrompt
 from pydantic import BaseModel, validator
-from watson_ai_planning.data_model.planning_types import PlanningResult
+from server.planners.drivers.planner_driver_datatype import PlanningResult
+from dataclasses import dataclass
```

Comment:
@jkeskingvillage We have ended up with a mixture of TypedDict, Pydantic, and dataclasses 🙈 let's settle on one? On the one hand, FastAPI is built on Pydantic but dataclasses are faster with it (allegedly) and [is supported natively](https://fastapi.tiangolo.com/advanced/dataclasses/). Also to consider, we need to migrate Pydantic to v2 anyway. On the other hand, Pydantic [does also support dataclasses as well](https://docs.pydantic.dev/latest/concepts/dataclasses/). Make a choice, I can do the migration or removal as the case may be.

Reply:
Yeah... Those are data types from planners. Let me try pydantic's dataclass. Good catch.